tldr/cross_file_calls/parsers/go.py

```
@@ -0,0 +1,246 @@
+            # Go function call patterns
+            patterns = [
+                # Function()
+                (r'(\b[A-Z]\w*)\s*\(', 'exported'),
+                # package.Function()
+                (r'(\b[a-z]\w+\.[A-Z]\w*)\s*\(', 'package'),
+                # method()
+                (r'(\b[a-z]\w+)\s*\(', 'local'),
+            ]
+            
+            for line_num, line in enumerate(content.split('\n'), 1):
+                for pattern, call_type in patterns:
```

Ruff (0.14.11)

Loop control variable `call_type` not used within loop body

Rename unused `call_type` to `_call_type`

(B007)